### PR TITLE
Log Ryujinx Version, OS Name, CPU Name and RAM size

### PR DESCRIPTION
**Pull request by Xpl0itR**
Windows:
image

"Linux":
image

### CODE REVIEW

File: Ryujinx/Program.cs [hunk 1]
```diff
@@ -108,5 +116,80 @@ private static void Glib_UnhandledException(GLib.UnhandledExceptionArgs e)
...
+                    {
+                        FileName               = "wmic",
+                        Arguments              = "cpu get Name /Value",
+                        RedirectStandardOutput = true
+                    }))
+                    {
+                        return process.StandardOutput.ReadToEnd().Trim().Split("=")[1];
+                    }
+                }
+                else
```
Comment (Thog) [inline Ryujinx/Program.cs:136]: You should maybe match "Unix" here instead and put a warning for unknown platforms

File: Ryujinx/Program.cs [hunk 2]
```diff
@@ -108,5 +116,80 @@ private static void Glib_UnhandledException(GLib.UnhandledExceptionArgs e)
...
+                    {
+                        FileName               = "wmic",
+                        Arguments              = "OS get TotalVisibleMemorySize /Value",
+                        RedirectStandardOutput = true
+                    }))
+                    {
+                        return Math.Round(double.Parse(process.StandardOutput.ReadToEnd().Trim().Split("=")[1]) / 1024, 0);
+                    }
+                }
+                else
```
Comment (Thog) [inline Ryujinx/Program.cs:176]: Same comment here

File: Ryujinx/Program.cs [hunk 3]
```diff
@@ -42,6 +45,11 @@ static void Main(string[] args)
             // Initialize Discord integration
             DiscordIntegrationModule.Initialize();
 
+            Logger.PrintInfo(LogClass.Application, $"Ryujinx Version: {Version}");
+            Logger.PrintInfo(LogClass.Application, $"Operating System: {RuntimeInformation.OSDescription} ({RuntimeInformation.OSArchitecture})");
+            Logger.PrintInfo(LogClass.Application, $"CPU: {GetCpuName()}");
+            Logger.PrintInfo(LogClass.Application, $"Total RAM: {GetRamSizeMb()} MB");
```
Comment (Thog) [inline Ryujinx/Program.cs:51]: Print "Unknown" in the case of ``GetRamSizeMb() == 0``

**Comment by Xpl0itR** [comment]
done

**Comment by tellowkrinkle** [comment]
If you want a macOS version
```cs
[DllImport("libSystem")]
static internal extern int sysctlbyname([MarshalAs(UnmanagedType.LPStr)] string name, out Int64 oldp, ref UIntPtr oldlenp, IntPtr newp, UIntPtr newlen);
[DllImport("libSystem")]
static internal extern int sysctlbyname([MarshalAs(UnmanagedType.LPStr)] string name, byte[] oldp, ref UIntPtr oldlenp, IntPtr newp, UIntPtr newlen);

if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX)) {
	UIntPtr len = (UIntPtr)sizeof(Int64);
	if (0 == sysctlbyname("hw.memsize", out Int64 mem, ref len, IntPtr.Zero, UIntPtr.Zero)) {
		RamSize = $"{mem / (1024*1024)} MB";
	}
	byte[] name = new byte[128];
	len = (UIntPtr)128;
	if (0 == sysctlbyname("machdep.cpu.brand_string", name, ref len, IntPtr.Zero, UIntPtr.Zero)) {
		CpuName = Encoding.UTF8.GetString(name);
	}
}
```

**Comment by Xpl0itR** [comment]
Resolved conflict

**Pull request merged**